### PR TITLE
biosample documentation update

### PR DESCRIPTION
**Pull request by pnrobinson**
@julesjacobsen Do you happen to have a complete biosample as JSON that we could use here? I copied this from the example, but some fields are empty

### CODE REVIEW

File: docs/biosample.rst [hunk 1]
```diff
@@ -13,30 +13,145 @@ Several instances (e.g. technical replicates) or types of experiments (e.g.
...
+     - true or false
+     - optional (default: false)
+     - whether the sample is being used as a normal control
+
+
+
+
+**Example**
+
+.. parsed-literal::
```
Comment (julesjacobsen) [inline docs/biosample.rst]: `.. parsed-literal::` should be `.. code-block:: json`

File: docs/biosample.rst [hunk 2]
```diff
@@ -13,30 +13,145 @@ Several instances (e.g. technical replicates) or types of experiments (e.g.
...
-        OntologyClass tumor_progression = 11;
-        OntologyClass tumor_grade = 12;
-        repeated OntologyClass tumor_stage = 13;
-        repeated OntologyClass diagnostic_markers = 14;
-        Procedure procedure = 15;
-        repeated HtsFile hts_files = 16;
-        repeated Variant variants = 17;
-        bool is_control_sample = 18;
-    }
+
```
Comment (julesjacobsen) [inline docs/biosample.rst]: Add **Data model**

**Comment by pnrobinson** [comment]
changed it.

File: docs/biosample.rst [hunk 3]
```diff
@@ -12,31 +12,146 @@ sequencing or a protein fraction from a gradient centrifugation.
...
+   * - hts_files
+     - :ref:`rstfile`
+     - optional
+     - list of high-throughput sequencing files derived from the biosample
+   * - variants
+     - :ref:`rstvariant`
+     - optional
+     - List of variants determined to be present in the biosample
+   * - bool is_control_sample
+     - true or false
```
Comment (julesjacobsen) [inline docs/biosample.rst]: `true or false`  -> `boolean`

Comment (pnrobinson) [reply docs/biosample.rst]: (blushing) I have changed to boolean :-0

File: docs/biosample.rst (hunk 1 above)
Comment (pnrobinson) [reply docs/biosample.rst]: done

File: docs/biosample.rst (hunk 3 above)
Comment (julesjacobsen) [reply docs/biosample.rst]: I mean, really! (rolls eyes)

**Pull request merged**